src/Statistics.h

```diff
@@ -1,117 +1,9 @@
 
 // Statistics.h
 
-
-
-
 #pragma once
 
-
-
-
-// tolua_begin
-enum eStatistic
```

Comment:
Should keep this enum here; it's exported to Lua. Instead, assign each line to its equivalent in the new `enum class Statistic` enum.

Reply:
does the export care if i change something?

the normal (non class) enum doesn't help me because i can't convert to the Statistics enum

Reply:
Ok - had a look in the binding - it exports the updated version

Note for myself: I changed some names to match the ones used by Minecraft - gotta add the to the lookup to any call using the enum

Reply:
I put the enum back again. added some conversion that you may still use the values from the old enum and added the new ones to the API too

Reply:
see later comments on removing it totaly of the API